workshop/01-link-datasource.md

```diff
@@ -0,0 +1,56 @@
+# Link your persistent datasource
+
+We're going to be setting up model objects to be transferred across our API, but to keep track of them, we'll need a persistent datasource to stash them in.
+
+Go to your command line in the root directory we've been working in, type `lb datasource`, and you'll see the following:
```

Comment:
I'd argue that that's not trivial at all! People will notice the difference, be confused/annoyed by it, and be distracted from the subject matter at hand! I'll fix it.